diracx-db/src/diracx/db/sql/job/db.py

```diff
@@ -132,7 +124,7 @@ async def setJobAttributes(self, job_id, jobData):
         stmt = update(Jobs).where(Jobs.JobID == job_id).values(jobData)
         await self.conn.execute(stmt)
 
-    async def _checkAndPrepareJob(
```

Comment:
Do we need this method to be public?

Reply:
It is used here

https://github.com/DIRACGrid/diracx/pull/296/files/1fb0f3aca4363c53eb1184880710f68baca40fba#diff-fdc6b06128306e37f88bb6e97b63e6bd65f3eec257ebce11d5dc33e6e3511c1eR140

Rescheduling could probably be further refactored, but it can't be moved inside jobdb because of other DB dependencies involved. Something that might be worth revisiting.

Reply:
Ah yes indeed sorry, I did not check that file yet